gulpfile.js/util/pattern-library/lib/addFileType.js

```diff
@@ -0,0 +1,18 @@
+var path = require('path')
+
+var addFileType = function (config, files) {
+  if (!Array.isArray(config.src)) {
```

Comment:
Do we have `config` at this point for sure? if so should we rely on a previous step to verify?